R/pkg/R/group.R

```diff
@@ -199,17 +199,10 @@ createMethods()
 #' Applies a R function to each group in the input GroupedData
 #'
 #' @param x a GroupedData
-#' @param func A function to be applied to each group partition specified by GroupedData.
-#'             The function `func` takes as argument a key - grouping columns and
-#'             a data frame - a local R data.frame.
-#'             The output of `func` is a local R data.frame.
-#' @param schema The schema of the resulting SparkDataFrame after the function is applied.
-#'               The schema must match to output of `func`. It has to be defined for each
-#'               output column with preferred output column name and corresponding data type.
-#' @return a SparkDataFrame
 #' @rdname gapply
 #' @name gapply
 #' @export
+#' @seealso \link{gapplyCollect}
 #' @examples
 #' \dontrun{
```

Comment:
@felixcheung , I took the same examples what gapply has. I did it similar to dapply and dapplyCollect. dapply and dapplyCollect have the same examples too. I think @sun-rui preferred having those the same.